src/cascadia/TerminalApp/defaults.json

```diff
@@ -1,56 +1,94 @@
 // THIS IS AN AUTO-GENERATED FILE! Changes to this file will be ignored.
 {
-    "alwaysShowTabs": true,
     "defaultProfile": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
+
+    // Launch Settings
     "initialCols": 120,
     "initialRows": 30,
-    "requestedTheme": "system",
+    "initialPosition": ",",

```

Comment:
i'd very much prefer we don't do this: it's a useless value that doesn't educate the user at all.